package.json

```diff
@@ -1,17 +1,23 @@
 {
   "name": "token-lists",
-  "version": "1.1.8",
+  "version": "1.1.9",
   "description": "Manages custom token lists for Brave Wallet",
   "dependencies": {
     "@metamask/contract-metadata": "git+https://git@github.com/MetaMask/contract-metadata.git",
     "@solana/spl-token-registry": "git+https://git@github.com/brave/solana-token-list.git",
+    "imagemin": "^8.0.1",
+    "imagemin-pngquant": "^9.0.2",
     "nodejs-file-downloader": "^4.9.3",
     "qyu": "^0.4.7",
     "sharp": "^0.30.5",
     "svg-resizer": "git+https://git@github.com/brave/svg-resizer.git"
   },
+  "resolutions": {
+    "semver-regex": ">=4.0.5"
+  },
   "main": "scripts/index.js",
   "scripts": {
+    "preinstall": "npx npm-force-resolutions",
```

Comment:
do we need this, or can https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides be used instead?